apps/x509.c

```diff
@@ -292,9 +294,9 @@ int x509_main(int argc, char **argv)
     int fingerprint = 0, reqfile = 0, checkend = 0;
     int informat = FORMAT_UNDEF, outformat = FORMAT_PEM, keyformat = FORMAT_UNDEF;
     int next_serial = 0, subject_hash = 0, issuer_hash = 0, ocspid = 0;
-    int noout = 0, CA_createserial = 0, email = 0;
+    int noout = 0, CA_createserial = 0, email = 0, multi = 0;
```

Comment:
For consistency, better place `multi` in the same order as used above, i.e., directly after `modulus`.

Reply:
Updated.